py-polars/tests/unit/functions/range/test_date_range.py

```diff
@@ -141,10 +141,26 @@ def test_date_ranges_single_row_lazy_7110() -> None:
 def test_date_range_end_of_month_5441() -> None:
     start = date(2020, 1, 31)
     stop = date(2021, 1, 31)
-    with pytest.raises(
-        pl.ComputeError, match=r"cannot advance '2020-01-31 00:00:00' by 1 month\(s\)"
-    ):
-        pl.date_range(start, stop, interval="1mo", eager=True)
+    result = pl.date_range(start, stop, interval="1mo", eager=True)
+    expected = pl.Series(
+        "date",
+        [
+            date(2020, 1, 31),
+            date(2020, 2, 29),
+            date(2020, 3, 29),
+            date(2020, 4, 29),
+            date(2020, 5, 29),
+            date(2020, 6, 29),
+            date(2020, 7, 29),
+            date(2020, 8, 29),
+            date(2020, 9, 29),
+            date(2020, 10, 29),
+            date(2020, 11, 29),
+            date(2020, 12, 29),
+            date(2021, 1, 29),
+        ],
+    )
```

Comment:
This seems like undesirable behavior to me. Instead of doing incremental steps of 1 month, we should be doing offsets of `1mo`, `2mo`, etc. That will result in a correct range where each entry is the last day of the month.

Could you make an issue for this?

Reply:
sure, I've reopened https://github.com/pola-rs/polars/issues/5441, which is pointed to in the title of this test (`test_date_range_end_of_month_5441`)